tests/test_http_parser.py

```diff
@@ -117,6 +117,20 @@ def test_parse_headers(parser: Any) -> None:
     assert not msg.upgrade
 
 
+def test_parse_headers_longline(parser: Any) -> None:
+    text = (
+        b"""GET /test HTTP/1.1\r
```

Comment:
Why is it `\r` and not `\r\n`

Reply:
Oh.. that `\n` is probably represented by the actual line feed in the literal :exploding_head:

Reply:
Yes! I saw it's done similarly in other tests as well.